src/flycapture2.pyx

```diff
@@ -50,12 +51,17 @@ def get_library_version():
 
 cdef class Context:
     cdef fc2Context ctx
-
-    def __cinit__(self):
-        cdef fc2Error r
-        with nogil:
-            r = fc2CreateContext(&self.ctx)
-        raise_error(r)
+    def __cinit__(self, context_type='USB'):
```

Comment:
Please mirror the underlying API closely. There is no need to be smart here. `gige_context=False` etc.